### ISSUE TITLE
consider lib-sass of ruby sass (it isn't as painfully slow)

### ISSUE THREAD

**Issue by stefanpenner** [chancancode/hn-reader#5]

**Comment by chancancode** [comment]
Yeah, I switched at some point because latest bourbon required some new sass features. I don't know what they need it for, but I probably ended up using 0.01% of bourbon, so maybe I can just get rid of it.

**Comment by chancancode** [comment]
I think I did something else wrong, overtime the rebuild got super slow, I haven't had time to investigate yet, but I am guessing it might have to do with the fixtures :(

**Comment by stefanpenner** [comment]
ruby-sass is amazingly slow

**Comment by stefanpenner** [comment]
I will muck with it during the morning talks :P 

@chancancode we should try to get this working with FastBoot ™. Once all those pieces are together, this would make an amazing HN submission.

**Comment by chancancode** [comment]
Yeah, I actually don't really read Hacker News much, so the only way for this project to not die is to use it as a learning vehicle and discussion point with people :P I would love to be taught how I could have done things better, or use it to experiment with things like fast boot, betas, etc.

(I didn't have time to upgrade cause I was in a rush to get this done, but now I can do it!)

**Comment by stefanpenner** [comment]
> Yeah, I actually don't really read Hacker News much,

ya, i do it rarely now as well.

> so the only way for this project to not die is to use it as a learning vehicle and discussion point with people :P

yes, this would make a great example app.

> I would love to be taught how I could have done things better, or use it to experiment with things like fast boot, betas, etc.

as i have time, I will try to be of help.
